polynote-kernel/src/test/scala/polynote/kernel/interpreter/python/PythonInterpreterSpec.scala

```diff
@@ -276,7 +288,35 @@ class PythonInterpreterSpec extends FreeSpec with Matchers with InterpreterSpec
           PythonObject.defaultReprs(test).toList should contain theSameElementsAs List(
             MIMERepr("text/plain", "Plaintext string"),
             MIMERepr("text/html", "<h1>HTML string</h1>"),
-            MIMERepr("application/x-latex", "latex{string}")
+            MIMERepr("application/x-latex", "latex{string}"),
+            MIMERepr("image/svg+xml", "<svg />"),
+            MIMERepr("image/jpeg", "somekindofbase64encodedjpeg"),
+            MIMERepr("image/png", "iguessabase64encodedpng"),
+            MIMERepr("application/x-blahblah", "blahblah")
+          )
+      }
+    }
+
+    "handle case where _repr_mimebundle_ returns a tuple" in {
+      val code =
+        """class Example(object):
+          |  def __init__(self):
+          |    return
+          |
+          |  def __repr__(self):
+          |    return "Plaintext string"
+          |
+          |  def _repr_mimebundle_(self, include=None, exclude=None):
+          |    return ({ "application/x-blahblah": "blahblah" }, {})
```

Comment:
Similar to https://github.com/polynote/polynote/pull/807/files#r388020264, from https://ipython.readthedocs.io/en/stable/config/integrating.html#MyObject._repr_mimebundle_ 

> _repr_mimebundle_(include=None, exclude=None)
>> Should return a dictionary of multiple formats, keyed by mimetype, or a tuple of two dictionaries: data, metadata.